community/kernel/src/main/java/org/neo4j/unsafe/impl/batchimport/ParallelBatchImporter.java

```diff
@@ -303,8 +304,8 @@ private void importRelationships( NodeRelationshipCache nodeRelationshipCache,
 
         long nextRelationshipId = 0;
         Configuration relationshipConfig = withBatchSize( config,
-                neoStore.getRelationshipStore().getRecordsPerPage() );
-        Configuration nodeConfig = withBatchSize( config, neoStore.getNodeStore().getRecordsPerPage() );
+                neoStore.getRelationshipStore().getRecordsPerPage() * 100 );
+        Configuration nodeConfig = withBatchSize( config, neoStore.getNodeStore().getRecordsPerPage() * 100 );
```

Comment:
Didn't think that got it in. I'll split that out to another commit and clean that up